### DISCUSSION TITLE
Add event log.

### DISCUSSION THREAD

**Discussion by ithakaa** [lxdware/lxd-dashboard#56] [General]
Matthew, I firstly want to thank you for an amazing tool, great work

A really nice feature of proxmox is the event log at the bottom of the page that can be hidden

Can we get something similar please?

**Comment by matthewalanpenning** [comment]
Thank you! I will have to take a look at what Proxmox does. I do try to write to the console log a lot of events that happen on the page (right-click and inspect the page). Also there is an event log database (select user icon at the top right and select logs) that may provide what you are looking for. Is there anything specific you would like to see in the logs?

**Reply by ithakaa** [reply]
yes, I did see the logs but do think it's more useful as a tail at the bottom of the page.
https://pve.proxmox.com/wiki/File:Screen-vm-snapshots.png
